src/utilities/FeatureTable.js

```diff
@@ -141,6 +141,41 @@ export class FeatureTable {
 
 	}
 
+	isDraco() {
+
+		const { extensions } = this.header;
+		return extensions && !! extensions[ '3DTILES_draco_point_compression' ];
+
+	}
+
+	getDracoBuffer() {
+
+		if ( ! this.isDraco() ) {
+
+			throw new Error( 'FeatureTable: Feature data is not draco encoded' );
+
+		}
+
+		const { buffer, binOffset } = this;
+
+		const { byteOffset, byteLength } = this.header.extensions[ '3DTILES_draco_point_compression' ];
+
+		return buffer.slice( binOffset + byteOffset, binOffset + byteOffset + byteLength );
+
+	}
+
+	getDracoProperties() {
+
+		if ( ! this.isDraco() ) {
+
+			throw new Error( 'FeatureTable: Feature data is not draco encoded' );
+
+		}
+
+		return this.header.extensions[ '3DTILES_draco_point_compression' ].properties;
+
+	}
+
```

Comment:
I think ideally the `FeatureTable` class wouldn't have to change since this is all really just relevant to the points properties, right? Otherwise if there are other relevant compressed buffers in the FeatureTable then it might make most sense to make a derivative class that abstracts away the draco decompression and automatically decompresses the data when a user calls "getData", for example.

Reply:
I've now moved more of the logic to a `PNTSFeatureTable` to keep the draco related logic away from other data types.

I wasn't able to abstract it fully so that you could just use `getData()` as normal. This was due to the `dracoLoader` returning geometry not just a typed array.